lib/stores/S3Store.js

```diff
@@ -75,21 +68,28 @@ class S3Store extends DataStore {
 
         delete options.partSize;
 
-        this.client = new aws.S3(Object.assign({}, {
-            apiVersion: '2006-03-01',
-            region: 'eu-west-1',
-        }, options));
+        this.client = new aws.S3(
+            Object.assign(
+                {},
+                {
+                    apiVersion: '2006-03-01',
+                    region: 'eu-west-1',
+                },
+                options
+            )
+        );
 
         log('init');
     }
 
     /**
-     * Check if the bucket exists in S3.
-     *
-     * @return {Promise}
-     */
+   * Check if the bucket exists in S3.
+   *
+   * @return {Promise}
+   */
```

Comment:
Regarding the formatting: The comment's indentation is one space less than for the method below. This looks a bit odd. Is this intended and enforced by the linter? Not a big deal, just wondering.

Reply:
I currently don't care about formatting or stylistic linting rules because after this PR I will start again in TS with a completely different Eslint setup and formatting with Prettier.